### DISCUSSION TITLE
confused about observable tests

### DISCUSSION THREAD

**Discussion by jmls** [avajs/ava#2772] [Q&A]
I have followed the example of using an observable in a test. but am getting nowhere. 

```
test('afterInsert', (t) => {
    t.plan(1);

    const test = query.afterCreateEvent$.pipe(
        tap((result) => {
            console.log('xxxxxxxxxxxxxx', result);
            t.pass();
         })
    );

    return test;
});
```

when I run this test. I get the "xxxxxx"  but the test seems to time out

What am I missing ?

**Comment by novemberborn** [comment]
I think AVA may be waiting for the observable itself to complete.

**Comment by jmls** [comment]
yup. thanks ;)
